class Exception4
{
 public static void main(String args[])
  {
   int age=Integer.parseInt(args[0]);
   System.out.println(age);
   System.out.println("Task completed");
  }
}